src/Nfts/ImageGallery/ImageGallery.tsx

```diff
@@ -30,7 +31,13 @@ export const ImageGallery = ({nfts = [], onSelect, onRefresh, isRefreshing}: Pro
       data={nfts}
       onRefresh={onRefresh}
       refreshing={isRefreshing}
-      renderItem={(nft) => <ModeratedImage onPress={() => onSelect(nfts.indexOf(nft))} nft={nft} key={nft.id} />}
+      renderItem={(nft) =>
+        MODERATING_NFTS_ENABLED ? (
```

Comment:
I think eslint is going to complain if MODERATING_NFTS_ENABLED === true is not used